packages/template-block/src/TemplateBlock.spec.ct.tsx

```diff
@@ -156,24 +167,23 @@ describe('Template Block', () => {
         });
 
         mount(<TemplateBlockWithStubs />);
-        cy.get(TEMPLATE_PREVIEW_SELECTOR).should('have.attr', 'src', expectedPreviewUrl);
+        cy.get(TEMPLATE_PREVIEW_IMAGE_SELECTOR).should('have.attr', 'src', expectedPreviewUrl);
     });
 
-    it("should render a template's preview if preview mode is set to template", () => {
+    it('should not render preview, but CTA and text side-by-side if preview is set to none', () => {
         const templateDummy = getTemplateDummyWithPages();
-        const expectedPreviewUrl = templateDummy.previewUrl;
         const [TemplateBlockWithStubs] = withAppBridgeBlockStubs(TemplateBlock, {
-            editorState: true,
             blockTemplates: {
                 template: [templateDummy],
             },
             blockSettings: {
-                // preview: PREVIEW_MODE_TEMPLATE,
+                preview: PREVIEW_MODE_NONE,
             },
         });
 
         mount(<TemplateBlockWithStubs />);
-        cy.get(TEMPLATE_PREVIEW_SELECTOR).should('have.attr', 'src', expectedPreviewUrl);
+        cy.get(TEMPLATE_PREVIEW_IMAGE_SELECTOR).should('not.exist');
+        cy.get(TEMPLATE_BLOCK_TEXT_CTA_WRAPPER_SELECTOR).should('not.have.class', 'tw-flex-col');
```

Comment:
wouldn't test on classes directly, would test on the css property, so we could replace the CSS framework without touching the tests

same on other tests

Reply:
Damn, and I rewrote some tests to test the class 😂 Makes sense, more framework agnostic. I see what I can do. 👌